app/routes/url.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express')
+var router = express.Router()
+const Url = require('../models/ShortUrl');
+
+function validURL(myURL) {
+    var pattern = new RegExp('^(https?:\\/\\/)?'+ 
+    '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.?)+[a-z]{2,}|'+ 
+    '((\\d{1,3}\\.){3}\\d{1,3}))'+ 
```

Comment:
store the regex in a string and use that string as regex expression

Reply:
Okay.